asyncy/processing/Services.py

```
@@ -427,7 +429,11 @@ def smart_insert(cls, story, line, command_conf: dict, key: str, value,
         if int(response.code / 100) == 2:
             content_type = response.headers.get('Content-Type')
             if content_type and 'application/json' in content_type:
-                return ujson.loads(response.body)
+                output = command_conf.get('output')
+                body = ujson.loads(response.body)
+                ServiceContract.validate_output_properties(
```

So what I was going for the last few times was to check for all ContractViolations here, catch them and add all the context to them.
Here you have all the info that's helpful to the user: what service, what method, the entire body etc.

---

I have that in validate_output_properties story and line i can add to all messages. When I was testing story printed story = <asyncy.Stories.Stories object at 0x7fce05b3cd30>.

---

I can add a general try except around this call and throw AsyncyError?. I had done that in my first commit then I removed it. Please tell me how I should do that.

Lets decide on a way to fix final exception thing and what information to include in the errors, I have made the rest of the changes. Hopefully we can close after that. I will make that change tomorrow.

---

I can't catch all exceptions and throw a new one, a bit confused here. Will think about it and come up with a solution. Do suggest a way if you know one.